src/include/migraphx/op/mod.hpp

```diff
@@ -47,9 +47,10 @@ struct mod : binary<mod>
     {
         auto a           = base_attributes();
         a["commutative"] = false;
+        a["point_op"]    = "${function:fmod}((${function:remainder}(${0}, ${1})) + ${1}, ${1})";
         return a;
     }
-    std::string point_function() const { return "mod"; }
+    // std::string point_function() const { return "mod"; }
```

Comment:
Yea this function can be removed. Also `name()` can be removed as well since the `binary<>` base class will use the name of the class by default. This could be done for `fmod` as well.

Reply:
All good. Done